### ISSUE TITLE
Compile issues with Cargo and getty

### ISSUE THREAD

**Issue by MggMuggins** [redox-os/userutils#18]
I get a lot of mismatched type errors with `getty` when I compile the repo with `cargo build`. When userutils is built as a part of cookbook, then it builds and deploys fine. All the errors are either "expected usize, found i32" or "expected i32, found usize". Any thoughts/ideas?

**Comment by nobubby** [comment]
Can confirm this happens with me too. Spams my console a bit actually.

**Comment by MggMuggins** [comment]
Yeah, only get this when compiling with x86_64 on Linux, not `x86_64-unkown-redox`.
